src/Spd.Manager.Licence/CrcControllingMemberContract.cs

```diff
@@ -0,0 +1,62 @@
+﻿using MediatR;
+using Spd.Manager.Shared;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Spd.Manager.Licence;
+public abstract record CrcControllingMemberApp
+{
+    public int AccessCode { get; set; } //is it int or string?
```

Comment:
it should be a string, I am not sure request will contain accessCode, you can leave it here.